fastlane/lib/fastlane/actions/slack.rb

```diff
@@ -22,29 +22,34 @@ def self.run(options)
         require 'slack-notifier'
 
         options[:message] = self.trim_message(options[:message].to_s || '')
-        options[:message] = Slack::Notifier::LinkFormatter.format(options[:message])
+        options[:message] = Slack::Notifier::Util::LinkFormatter.format(options[:message])
 
-        notifier = Slack::Notifier.new(options[:slack_url])
+        if options[:channel].to_s.length > 0
+          channel = options[:channel]
+          channel = ('#' + notifier.channel) unless ['#', '@'].include?(channel[0]) # send message to channel by default
```

Comment:
Looks like `notifier` isn't assigned at this point yet, so I guess you can reproduce the issue when setting a `channel` parameter? 

That happened during `bundle exec fastlane bump` on the fastlane's Fastfile.

Reply:
Got it. All fixed!